MetamodelAlgemeen.md

```diff
@@ -307,18 +307,20 @@ Een object heeft kenmerken waarvoor gegevens bekend zijn.
 
 *Toelichting:*
 
-Voorbeelden van gegevens zijn de waardes ‘Jan’ en ‘1-1-1970’ betreffende de naam en de geboortedatum van een object van het type Persoon. 
+Voorbeelden van gegevens zijn: de naam van een persoon is 'Jan', de geboortedatum van Jan is ‘1-1-1970’. Eigenschappen van een object kun je waarnemen of beweren en vervolgens vastleggen als gegeven. Aan alleen de _waarde_ '1-1-1970' heb je niet zo veel als je niet weet dat het om de geboortedatum van een persoon gaat (de _eigenschap_), en om welke persoon het precies gaat (het _onderwerp_ van gesprek). Verder is het uiteraard van belang om te weten wat de betekenis is die bij het gegeven hoort. 
 
-Een informatiemodel specificeert niet de gegevens zelf. Een gegeven zoals '1-1-1970' noemen we een waarde voor een attribuut van Jan. In het informatiemodel wordt dit het attribuutsoort 'geboortedatum' of 'overlijdensdatum' van een objecttype Persoon genoemd. Het is dus ook van belang om welk kenmerk van Jan het gaat. Merk op dat een gegeven ‘1-1-1970’ geen informatie biedt zonder dat er eerst duidelijkheid is over het kenmerk c.q. het attribuutsoort 'geboortedatum' of 'overlijdensdatum' het gaat en om welke object van het objecttype Persoon het gaat. Het waargenomen feit dat twee objecten een relatie hebben wordt ook gezien als een een modelelement waarvoor een gegeven bekend kan zijn.
+Een model specificeert niet de gegevens zelf. Een gegeven zoals '1-1-1970' noemen we een waarde voor een attribuut van Jan. In het informatiemodel wordt dit bv. het attribuutsoort 'geboortedatum' of attribuutype 'overlijdensdatum' van een objecttype Persoon genoemd. Het is dus ook van belang om welke eigenschap van welke specifieke persoon het gaat. Dat twee objecten een relatie hebben wordt ook gezien als een een modelelement waarvoor een gegeven bekend kan zijn. 
 
-Een gegeven kan al bekend zijn zodra deze wordt waargenomen, maar in een informatievoorziening zijn gegevens de objectief waarneembare neerslag of registratie van feiten op een bepaald medium, zodanig dat deze gegevens uitgewisseld en voor langere tijd bewaard kunnen worden. Dat kan op papier, in digitale vorm, et cetera. Een gegeven in een informatievoorziening is dus een vastgelegde waarde voor een waargenomen feit dat toegekend is aan een kenmerk van een specifiek object. 
+Bewering: een uitspraak, die meestal als waar bedoeld is maar niet per definitie waar is. Mijn geboortedatum is '1-1-1970', de waarde van dit huis is 200.000 euro (in het jaar 2018), de sensor waarneming dat de auto met kenteken 'AA 1234' met een snelheid van 120 kilometer per uur reed op de A1 bij hectometeraanduiding 20.8 op tijdsmoment 16:00. Wie de bewering gedaan heeft is vaak relevant. Denk bij een bewering aan hetgeen waarover besloten wordt door een bevoegd gezag, of aan een persoon die gegevens opgeeft voor een bepaald doel. 
```

Comment:
> de sensor waarneming dat de auto met kenteken 'AA 1234' met een snelheid van 120 kilometer per uur reed op de A1 bij hectometeraanduiding 20.8 op tijdsmoment 16:00

Dit vind ik een lastig voorbeeld. Is het 1 gegeven? Zo ja, wat is het onderwerp, het kenmerk, en de waarde?

Reply:
Marco vond dit juist belangrijk om een voorbeeld met een context met tijd erin te noemen. 

Om de discussie te voorkomen zal ik alles achter 'op' weglaten, maar het is wel interessant om te analyseren of dit nou 1 of 2+ gegevens zijn. Zal apart issue hiervoor opvoeren.